ApnsPHP/Message/LiveActivity.php

```diff
@@ -0,0 +1,259 @@
+<?php
+
+/**
+ * This file contains the LiveActivity class.
+ *
+ * SPDX-FileCopyrightText: Copyright 2024 Move Agency Group B.V., Zwolle, The Netherlands
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+namespace ApnsPHP\Message;
+
+use ApnsPHP\Message;
+use AssertionError;
+use UnexpectedValueException;
+
+/**
+ * Live activity style message.
+ */
+class LiveActivity extends Message
+{
+    /**
+     * Content state information
+     * @var array|object
+     */
+    private array|object $state;
+
+    /**
+     * Attributes at the start of an activity
+     * @var array|object
+     */
+    private array|object $attributes;
+
+    /**
+     * Type of attributes
+     * @var string
+     */
+    private string $attributes_type;
+
+    /**
+     * Event for the activity
+     * @var LiveActivityEvent
+     */
+    private LiveActivityEvent $event;
+
+    /**
+     * Time when the activity will become stale
+     * @var int
+     */
+    private int $stale_time;
+
+    /**
+     * Time when the activity will dismiss itself
+     * @var int
+     */
+    private int $dismiss_time;
```

Comment:
If it's a timestamp, let's call it a timestamp. Under `time` I would expect a time string like "00:00"